docs/catalog/web/minify-web-assets.md

```diff
@@ -0,0 +1,34 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: web
+tags: 
+ - web-assets
```

Comment:
`web-assets` is not a supported tag for patterns. You can see a list of available tags at https://patterns.greensoftware.foundation/guide/suggested-tags/ but I think just adding `web` here would be enough.